### DISCUSSION TITLE
Monitor image that is not a container?

### DISCUSSION THREAD

**Discussion by AthrogatePEQ** [containrrr/watchtower#1225] [General] [answered]
Has anyone come up with a way to monitor images that are used as the base image to customize so that we would know when we have to rebuild an image.

For example, I use the node-red/nodered:latest as a base image that I add packages to for my specific needs.  I need to know when node-red is updated so that I can repull and re-build my custom image to deploy as a container.

**Comment by simonaronsson** [comment, marked as answer]
This is not something supported by watchtower. My suggestion would be to set up a GHA that rebuilds your docker image on a schedule and pushes it either to ghcr.io or to dockerhub so that you may let watchtower listen to it.
